types/test.ts

```diff
@@ -15,6 +15,7 @@ x('urlset', {xmlns}) // $ExpectType Element
 x('urlset', {xmlns}, 'string') // $ExpectType Element
 x('urlset', {xmlns}, ['string', 'string']) // $ExpectType Element
 x('urlset', {xmlns}, x('loc'), 'string') // $ExpectType Element
+x('urlset', {xmlns}, x('loc'), 100) // $ExpectType Element
```

Comment:
could we also add a test case for attributes?

```suggestion
x('urlset', {xmlns}, x('loc'), 100) // $ExpectType Element
x('urlset', {xmlns, num: 100, bool: true}, x('loc'), 100) // $ExpectType Element
```